src/analysis/special_functions/trigonometric.lean

```diff
@@ -1592,7 +1592,7 @@ end
 
 end angle
 
-/-- `real.sin` as an `order_iso` between `[-(π / 2), π / 2]` and `[-1, 1]`. -/
+/-- `real.sin` as an `order_iso` between `(-(π / 2), π / 2)` and `(-1, 1)`. -/
 def sin_order_iso : Icc (-(π / 2)) (π / 2) ≃o Icc (-1:ℝ) 1 :=
```

Comment:
I don't follow the docstring change -- isn't it indeed the closed interval?

Reply:
Thanks! I was merging an old branch with master, and chose the wrong version for this chunk.